sdk_container/src/third_party/coreos-overlay/coreos/config/env/net-dns/bind

```diff
@@ -0,0 +1,62 @@
+# Keep only tool binaries and libraries those binaries need.
+ndb_install_mask="
+        /etc
+        /var
+        /usr/bin/arpaname
+        /usr/bin/named-*
+        /usr/bin/nsec3hash
+        /usr/lib/systemd
+        /usr/lib/tmpfiles.d
+        /usr/lib64/bind
+        /usr/lib64/libisccc*
+        /usr/libexec
+        /usr/sbin
+"
```

Comment:
If it helps, I think `INSTALL_MASK` can be inverted with `* -/usr/bin/dig -/usr/lib*/*.so* …`.